addons/cppcheckdata.py

```diff
@@ -10,7 +10,6 @@
 import argparse
 from fnmatch import fnmatch
 import json
-import os
```

Comment:
I took the freedom to remove this import as part of this pr. I hope it's fine. Inside the finle is nowhere used.